src/main/groovy/org/jenkinsci/gradle/plugins/jpi/JpiExtension.groovy

```diff
@@ -146,7 +146,7 @@ class JpiExtension {
         }
 
         if (new VersionNumber(this.coreVersion) >= new VersionNumber('2.64')) {
-            testHarnessVersion = '2.31'
+            testHarnessVersion = '2.60'
```

Comment:
It looks like [jenkins-test-harness is declaring a dependency on Jenkins 2.60.3](https://github.com/jenkinsci/jenkins-test-harness/blob/master/pom.xml#L84-L100), and as such this change should be OK. Could you confirm this is the case @darxriggs?

I'm not very familiar with this part of plugin, but wanted to check on this since [Jenkins 2.64](https://jenkins.io/changelog-old/#v2.64) was released on 2017-06-04, but [jenkins-test-harness 2.60](https://github.com/jenkinsci/jenkins-test-harness/releases/tag/jenkins-test-harness-2.60) was just released last week.

Reply:
Yes, this should be compatible. I also checked this.

Reply:
Excellent, thanks for confirming and for the update!